airflow/dag_processing/bundles/base.py

```diff
@@ -56,7 +56,7 @@ def _dag_bundle_root_storage_path(self) -> Path:
 
         This is the root path, shared by various bundles. Each bundle should have its own subdirectory.
         """
-        if configured_location := conf.get("core", "dag_bundle_storage_path"):
+        if configured_location := conf.get("core", "dag_bundle_storage_path", fallback=None):
```

Comment:
Do you instead want to change the following default to `""`

https://github.com/apache/airflow/blob/33bf3ede9e7d7495b68dbdeef687ca4c53b1a2e2/airflow/config_templates/config.yml#L31-L38

Reply:
```py
In [1]: from airflow.configuration import conf

In [2]: conf.get("core", "dag_bundle_storage_path")
Out[2]: ''
```

Reply:
It would work, but it feels "less right" to me. I think the right answer is to fix it so None is valid default, but that's a problem for another day I think.